cmd/src/bin/tikv-ctl.rs

```diff
@@ -209,44 +209,45 @@ trait DebugExecutor {
                     let region_epoch = r.get_region_epoch();
                     let peers = r.get_peers();
                     json!({
-                "region": json!({
-                    "id": r.get_id(),
-                    "start_key": hex::encode_upper(r.get_start_key()),
-                    "end_key": hex::encode_upper(r.get_end_key()),
-                    "region_epoch": json!({
-                        "conf_ver": region_epoch.get_conf_ver(),
-                        "version": region_epoch.get_version()
-                    }),
-                    "peers": peers.iter().map(|p| json!({
-                            "id": p.get_id(),
-                            "store_id": p.get_store_id(),
-                            "role": format!("{:?}", p.get_role()),
-                        })).collect::<Vec<_>>(),
+                        "region": json!({
+                            "id": r.get_id(),
```

Comment:
verbose with `region_id`

Reply:
I think this is ok, because it's the original structure of `RegionLocalState", while `region_id` is the top-level identifier.